FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth/DemoTasks/MutualAuthMQTTExample.c

```diff
@@ -546,6 +544,9 @@ static TlsTransportStatus_t prvConnectToServerWithBackoffRetries( NetworkCredent
             pxNetworkCredentials->pAlpnProtos = AWS_IOT_MQTT_ALPN;
             pxNetworkCredentials->alpnProtosLen = AWS_IOT_MQTT_ALPN_LENGTH;
         #endif
+    #else
+        /* SNI needs to be disabled for an MQTT broker that has no hostname. */
```

Comment:
Use of "MQTT broker that has no hostname" can be misleading. test.mosquitto.org has a hostname..